internal/node/controller/init.go

```diff
@@ -0,0 +1,117 @@
+package controller
+
+import (
+	"path/filepath"
+
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/go-connections/nat"
+	types "github.com/eagraf/habitat-new/core/api"
+	"github.com/eagraf/habitat-new/core/state/node"
+	"github.com/eagraf/habitat-new/internal/node/config"
+	"github.com/eagraf/habitat-new/internal/node/constants"
+	"github.com/eagraf/habitat-new/internal/node/hdb"
+	"github.com/google/uuid"
+)
+
+func generatePDSAppConfig(nodeConfig *config.NodeConfig) types.PostAppRequest {
+
+	pdsMountDir := filepath.Join(nodeConfig.HabitatPath(), "pds")
+
+	// TODO @eagraf - unhardcode as much of this as possible
+	return types.PostAppRequest{
+		AppInstallation: &node.AppInstallation{
+			Name:    "pds",
+			Version: "1",
+			UserID:  constants.RootUserID,
+			Package: node.Package{
+				Driver: "docker",
+				DriverConfig: map[string]interface{}{
+					"env": []string{
+						"PDS_HOSTNAME=ethangraf.com",
+						"PDS_DATA_DIRECTORY=/pds",
+						"PDS_BLOBSTORE_DISK_LOCATION=/pds/blocks",
+						"PDS_PLC_ROTATION_KEY_K256_PRIVATE_KEY_HEX=5290bb1866a03fb23b09a6ffd64d21f6a4ebf624eaa301930eeb81740699239c",
```

Comment:
hmm should this key be in here?

Reply:
Yeah I don't want to spend too much time about getting the configuration of PDS perfect right now.